packages/kit/src/runtime/server/page/actions.js

```diff
@@ -110,19 +110,19 @@ function action_json(data, init) {
 
 /**
  * @param {import('types').RequestEvent} event
- * @param {import('types').SSRNode} leaf_node
  */
-export function is_action_request(event, leaf_node) {
-	return leaf_node.server && event.request.method !== 'GET' && event.request.method !== 'HEAD';
+export function is_action_request(event) {
+	const { method } = event.request;
+	return method !== 'GET' && method !== 'HEAD';
```

Comment:
On my phone so viewing the code a few lines at a time and could be misunderstanding, but shouldn't this be checking that method is POST?

Reply:
Was confused by that as well, but didn't want to trigger any [unwanted changes](https://xkcd.com/1172/) so I just kept it as is.

PATCH, DELETE don't ever reach this point but other unknown methods might, do you want to tighten it down?

Reply:
This is probably a reminiscent of old times, yes we can simplify that to just check for POST